### ISSUE TITLE
Image Picker - A screen reader doesn't announce the Upload Image, Delete Item and Add Image button captions properly

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-creator#5408]
T17636 - Accessibility 
https://surveyjs.answerdesk.io/internal/ticket/details/T17636
***
[View Demo](https://plnkr.co/edit/jrLfGDnaJ5WvODHb)

When tabbing though an image picker, the screen reader (NVDA) doesn't announce actual captions of the upload, delete item buttons (e.g., upload an image to an item, remove an item); also, the add image button is announced as 'section'. Instead, the reader says `section`.

Please refer to the following screencast:

https://github.com/surveyjs/survey-creator/assets/22372972/da94f17b-38ad-4438-b4c9-10021ce94ef5
